rice-middleware/impl/src/main/java/org/kuali/rice/kew/routelog/web/RouteLogAction.java

```diff
@@ -155,7 +157,12 @@ private ActionRequestValue switchActionRequestPositionIfPrimaryDelegatePresent(
     	 */
     	
     	if (!actionRequest.isRoleRequest()) {
-    		List<ActionRequestValue> primaryDelegateRequests = actionRequest.getPrimaryDelegateRequests();
+            /**
+             * KULRICE-14183 : Detaching the action request object so that re-ordering of action requests for primary delegates does not get incorrectly persisted
+             */
```

Comment:
I think the above comment should be changed so that it matches the comment show below.  The jira number can be removed and it can be changed to a line comment instead of a block comment.